addons/payment_stripe/controllers/main.py

```diff
@@ -51,4 +51,9 @@ def stripe_create_charge(self, **post):
         _logger.info('Stripe: entering form_feedback with post data %s', pprint.pformat(response))
         if response:
             request.env['payment.transaction'].sudo().form_feedback(response, 'stripe')
+        tx_transaction = request.env['payment.transaction'].browse(request.session.get('website_payment_tx_id', False))
```

Comment:
This is kind of weird, I'm not sure there is a way around it but it essentially means we are creating two payment.transaction records for one single payment of eg. an online quotation?